src/Hidden/features.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import {Meta} from '@storybook/react'
+import React from 'react'
+import {Hidden} from './Hidden'
+import {Box, Button} from '..'
+
+const meta: Meta = {
+  title: 'Drafts/Components/Hidden/Features',
+  component: Hidden,
+}
+export default meta
+
+export const HideContent = () => (
+  <Box>
+    <Hidden on={['narrow']}> This value is shown in regular and wide viewports</Hidden>
+    <Hidden on={['regular']}> This value is shown in narrow and wide viewports</Hidden>
+    <Hidden on={['wide']}> This value is shown in narrow and regular viewports</Hidden>
```

Comment:
For these common cases it might be nice to call out that you can pass a `string` and it should work as expected.

```suggestion
    <Hidden on="narrow"> This value is shown in regular and wide viewports</Hidden>
    <Hidden on="regular"> This value is shown in narrow and wide viewports</Hidden>
    <Hidden on="'wide"> This value is shown in narrow and regular viewports</Hidden>
```